django/django-quiz.md

```diff
@@ -1,333 +1,333 @@
     	class Constellation(models.Model):
     		stars = models.ManyToManyField(Star)
 
-    - 	class Star(models.Model):
+    - [] 	class Star(models.Model):
     		constellation = models.ForeignKey(Constellation, on_delete=models.CASCADE)
     	class Constellation(models.Model):
     		stars = models.ForeignKey(Star, on_delete=models.CASCADE)
 
-    - 	class Star(models.Model):
+    - [] 	class Star(models.Model):
     		name = models.CharField(max_length=100)
     	class Constellation(models.Model):
     		stars = models.OneToManyField(Star)
 
-    - 	class Star(models.Model):
+    - [] 	class Star(models.Model):
     		constellation = models.ManyToManyField(Constellation)
     	class Constellation(models.Model):
     		name = models.CharField(max_length=100)
 
-Q18. Which is NOT a valid step in configuring your Django 2.x instance to serve up static files such as images or CSS?
+#### Q18. Which is NOT a valid step in configuring your Django 2.x instance to serve up static files such as images or CSS?
 
-    👍 - In your urls file, add a pattern that includes the name of your static directory.
-    - Create a directory named static inside your app directory.
-    - Create a directory named after the app under the static directory, and place static files inside.
-    - Use the template tag {percentage mark load static percentage mark}.
+    - [x] - In your urls file, add a pattern that includes the name of your static directory.
+    - [] Create a directory named static inside your app directory.
+    - [] Create a directory named after the app under the static directory, and place static files inside.
+    - [] Use the template tag {percentage mark load static percentage mark}.
 
-Q19. What is the correct way to make a variable available to all of your templates?
+#### Q19. What is the correct way to make a variable available to all of your templates?
 
-    - Set a session variable.
-    - Use a global variable.
-    - Add a dictionary to the template context.
-    👍 - Use RequestContext.
+    - [] Set a session variable.
+    - [] Use a global variable.
+    - [] Add a dictionary to the template context.
+    - [x] - Use RequestContext.
 
-Q20. Should you create a custom user model for new projects?
+#### Q20. Should you create a custom user model for new projects?
 
-    - No. Using a custom user model could break the admin interface and some third-party apps.
-    - Yes. It is easier to make changes once it goes into production.
-    👍 - No. Django's built-in models.User class has been tried and tested—no point in reinventing the wheel.
-    - Yes, as there is no other option.
+    - [] No. Using a custom user model could break the admin interface and some third-party apps.
+    - [] Yes. It is easier to make changes once it goes into production.
+    - [x] - No. Django's built-in models.User class has been tried and testedÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½no point in reinventing the wheel.
```

Comment:
please double-check why there are special characters instead of alphabetical symbols